templates/dashboard/collection/modal_delete.html

```diff
@@ -0,0 +1,22 @@
+{% load i18n %}
```

Comment:
We have a base modal for such cases: `dashboard/base_modal.html`. Also, we have a convention of naming those templates as `confirm_delete.html`.